code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/MessagingExtension.java

```diff
@@ -161,7 +161,7 @@ protected void onRegistered() {
                             "MessagingEvents",
                             event -> {
                                 if (InternalMessagingUtils.isGetPropositionsEvent(event)) {
-                                    edgePersonalizationResponseHandler.retrieveMessages(
+                                    edgePersonalizationResponseHandler.retrieveCachedContentCards(
```

Comment:
`getPropositionsForSurfaces` is also used to get the cached code based propositions. `retrieveCachedContentCards` seems to only be returning the content cards. Are we introducing a new API for getting CBE content?

Reply:
updated